frontend/src/utils/constants.js

```diff
@@ -84,6 +84,19 @@ export const SUPPLEMENTARY_TYPES = Object.freeze({
   TRANSPORT: 3,
 })
 
+export const REQUEST_CATEGORY_TYPES = Object.freeze({
+  ACCOUNT_MAINTENANCE: 'a640abcf-7377-ee11-8179-000d3a09d132',
+})
+
+export const REQUEST_SUB_CATEGORY_TYPES = Object.freeze({
```

Comment:
same for these ones. why don't we use the ids from lookup?

Reply:
"requestSubCategories": [
        {
            "categoryName": "Organization Details",
            "subCategoryId": "af5d9441-bcdc-ee11-904d-000d3a0a18e7"
        },
        {
            "categoryName": "Facility Details",
            "subCategoryId": "bf585d4e-bcdc-ee11-904d-000d3a0a18e7"
        },
        {
            "categoryName": "Add/change a licence",
            "subCategoryId": "4f612456-bcdc-ee11-904d-000d3a0a18e7"
        },
        {
            "categoryName": "Other",
            "subCategoryId": "49caab5c-bcdc-ee11-904d-000d3a0a18e7"
        },
        {
            "categoryName": "Organization phone/email",
            "subCategoryId": "3f8eaec3-8fdd-ee11-904d-000d3a0a18e7"
        },
        {
            "categoryName": "Facility phone/email",
            "subCategoryId": "1767cfc9-8fdd-ee11-904d-000d3a0a18e7"
        }
    ],

Reply:
Fixed.

Reply:
oh dear! i forgot to transition from constants. Please see how i'm doing it now.